deployment/operator/README.md

```diff
@@ -40,12 +40,12 @@ spec:
   values:
     nri:
       plugin:
-        index: 90
+        index: 90                       # optional
       runtime:
-        patchConfig: true
-#       config:
-#         pluginRegistrationTimeout: 5s
-#         pluginRequestTimeout: 2s
+        patchConfig: true               # optional
+      config:
```

Comment:
I think this is tabulated incorrectly. The value key for that config is `values.nri.runtime.config`, not `values.nri.config`. So it should be at the same level as `patchConfig`, non ?